ideascube/configuration/registry.py

```diff
@@ -12,19 +12,29 @@
 REGISTRY = {
     # 'namespace': {
     #     'key': {
+    #         'summary': 'Help text for the management command',
+    #         'pretty_type': 'Pretty type(s) for the management command',
     #         'type': str,
     #         'default': 'Default value',
     #     },
     # },
     'server': {
         'site-name': {
+            'summary': 'The pretty name of the server, as seen by users of '
+                       'the web interface.',
+            'pretty_type': 'A string',
             'type': str,
             'default': getattr(settings, 'IDEASCUBE_NAME', 'Ideas Cube'),
         },
     },
 }
 
 
+def get_all_namespaces():
+    for namespace in sorted(REGISTRY):
+        yield namespace
+
```

Comment:
It was a suggestion to use `yield from sorted(REGISTRY)` instead of the `for` loop.

But it's just a suggestion. Few people use it.
The only "massive" use of `yield from` I know about was with asyncio before `await` came into python.